1908-01-20.xml

```diff
@@ -0,0 +1,1344 @@
+                    31029-13-3</p>
+                     </div>
+                    <!-- Image advert for Magna Cider -->
+                    <div type="advert" xml:id="deg-ad-bsl01">
+                        <head>BERLITZ SCHOOLS OF LANGUAGES</head>
+                        <p>236 BRANCHES.</p>
+                        <p>French, German, Italian, Greek, Arabic, etc.</p>
+                        <p>Private Lessons, Residence Lessons, taught by Native Masters </p>
+                        <p>ALEXANDRIA: 26 Rue Rosette (close to Zizinia Theatre.)</p>
+                        <p>CAIRO : 1 Sharia Kamel. </p>
+                        <p>TRIAL LESSONS FREE</p>
+                    </div>
+                
+                <div type="item" xml:id="deg-el-egsa02">
+                    <head>THE EGYPTIAN GAZETTE.</head>
+                    <p>SUBSCRIPTIONS.—Alexandria, Cairo, and the Interior of Egypt (including
+                        delivery in Alexandria or postage to subscriber's address) P.T. 231½ per
+                        annum, P.T. 116 for six months, P.T. 80 for three months. To other countries
+                        in the Postal Union P.T. 273 (£2.16s.) per annum. Six months P.T. 136½
+                        (£1.8s.), three months P.T. 92 (£0.19s.) N.B.—Subscriptions commence from
+                        the 1st or 16th of any month. </p>
+                    <p>ADVERTISEMENTS.—P.T. 4 per line. Minimum charge P.T. 20. Births, Marriages,
+                        or Deaths, not exceeding three lines, P.T. 20. Every additional line P.T.
+                        10. Notices in news column P.T. 20 per line. Contracts entered into for
+                        standing advertisements. </p>
+                    <p>ADVERTISEMENTS and SUBSCRIPTIONS are due in advance. P.O. Orders and Cheques
+                        to be made payable to the Editor and Manager, Rowland Snelling, Alexandria. </p>
+                    <p>London Offices : 36, New Broad-street. B.C. </p>
+                    <p>THE EGYPTIAN GAZETTE can be obtained in London at our office, 36, New Broad
+                        Street, E.C.</p>
+                    <p>Cairo Offices.-No. 1 Sharia Vervudachi, (opposite the Agricultural Bank.)</p>
+                    <p>THE "EGYPTIAN GAZETTE” IS PRINTED ON PAPER MANUFACTURED AND SUPPLIED BY THE
+                        LONDON PAPER MILLS Co., LIMITED (SALES OFFICE: 27, CANNON STREET, E.C.) </p>
+                </div>
+       
+                        <div type="item" feature="masthead" xml:id="deg-el-egpi01">
+                            <head>The Egyptian Gazette</head>
+                            <p>An English Daily Newspaper, Established in 1880.</p>
+                            <p>Editor &amp; Manager: R. Snelling.</p>
+                            <p>Price: One Piastre Tariff.</p>
+                            <p><date when="1905-07-13">Monday, January 20, 1908</date>.</p> 
+                            <div type="item">
```

Comment:
You need a `</div>` on this line, to close the masthead. As it's currently formatted, the article on "The Future" is inside the masthead.